src/wayland/shm/pool.rs

```diff
@@ -44,7 +44,7 @@ static DROP_THIS: Lazy<SyncSender<InnerPool>> = Lazy::new(|| {
     tx
 });
 
-thread_local!(static SIGBUS_GUARD: Cell<(*const MemMap, bool)> = Cell::new((ptr::null_mut(), false)));
+thread_local!(static SIGBUS_GUARD: Cell<(*const MemMap, bool)> = const { Cell::new((ptr::null_mut(), false)) });
```

Comment:
This changes the MSRV?

Reply:
Not 100% sure, but I don't think so: this is 3 years old https://github.com/dtolnay/rust/commit/c6eea222a9cafeeeedfa1767232328710b350447